amethyst_rendy/src/plugins.rs

```diff
@@ -0,0 +1,310 @@
+//! Set of predefined implementations of `RenderPlugin` for use with `RenderingBundle`.
+
+use crate::{
+    bundle::{
+        ImageOptions, OutputColor, RenderOrder, RenderPlan, RenderPlugin, Target, TargetPlanOutputs,
+    },
+    pass::*,
+    sprite_visibility::SpriteVisibilitySortingSystem,
+    visibility::VisibilitySortingSystem,
+    Backend, Factory, Format, Kind,
+};
+use amethyst_config::Config;
+use amethyst_core::{
+    ecs::{DispatcherBuilder, ReadExpect, Resources, SystemData},
+    SystemBundle,
+};
+use amethyst_error::Error;
+use amethyst_window::{DisplayConfig, ScreenDimensions, Window, WindowBundle};
+use palette::Srgb;
+use rendy::{
+    graph::render::RenderGroupDesc,
+    hal::command::{ClearColor, ClearDepthStencil, ClearValue},
+};
+use std::path::Path;
+
+/// A [RenderPlugin] for opening a window and displaying a render target to it.
+///
+/// When you provide [DisplayConfig], it opens a window for you using [WindowBundle].
+#[derive(Default, Debug)]
+pub struct RenderToWindow {
```

Comment:
It can be worth splitting this to a separate crate (or putting behind a feature gate) for server type applications that may want to render to a frame buffer and send it through a video encoder stream, but not a window. Not a hard comment (just resolve if out of scope).

Reply:
Yeah i was considering such thing. Then i realised we already pull `amethyst_window` as a dependency anyway... :P I just kept it here due to end-user convinience, but definitely will put it under a feature flag now.